framework/encode/vulkan_capture_manager.cpp

```diff
@@ -1137,24 +1170,27 @@ VulkanCaptureManager::OverrideCreateRayTracingPipelinesKHR(VkDevice
                                                             createInfoCount,
                                                             pCreateInfos_unwrapped,
                                                             pAllocator,
-                                                            output_piplines);
+                                                            pPipelines);
     }
 
     if (((result == VK_SUCCESS) || (result == VK_OPERATION_DEFERRED_KHR) ||
          (result == VK_OPERATION_NOT_DEFERRED_KHR)) &&
         (pPipelines != nullptr))
     {
+        if (deferredOperation != VK_NULL_HANDLE &&
+            ((result == VK_OPERATION_DEFERRED_KHR) || (result == VK_OPERATION_NOT_DEFERRED_KHR)))
+        {
+            ProcessDeferredOperation(device, deferredOperation);
```

Comment:
This changed target title behavior because the deferred operation join call come from GFXR, not original application.